src/openrct2/config/Config.h

```diff
@@ -204,6 +204,7 @@ struct FontConfiguration
 struct PluginConfiguration
 {
     bool enable_hot_reloading;
+    std::string allowed_hosts;
```

Comment:
Usage would be a lot clearer if we stored this as a container somehow, instead of a comma separated list, but I'm not sure if this is possible (may be a limitation of the serialiser)

We might want to store it as a container and "split" on load, because at least that will make querying easier.

Reply:
I suppose it would be possible to add a global vector with whitelisted IPs, to be loaded/saved from the config.

Since IntelOrca is okay with the current implementation, I think that that is best left for another PR.